File: swagger/models/get_client_account_balances_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetClientAccountBalancesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		clients
		A list of clients.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Clients': 'clients',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'clients': 'Clients',
		}

		'PaginationResponse': ('pagination_response', PaginationResponse),
	clients = None

File: swagger/Models/GetClientAccountBalancesResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetClientAccountBalancesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * A list of clients.
	 * 
	 * @var Client[] 
	 */
	public $clients;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Clients' => ['clients', 'array', Client::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'clients' => ['Clients', 'array', Client::class],
			];
	}
}
